scripts/docker/Dockerfile

```diff
@@ -1,27 +1,28 @@
-FROM ubuntu:xenial
+FROM debian:stretch-slim
 LABEL maintainer "devops-team@parity.io"
 LABEL description="Substrate: The platform for blockchain innovators"
 
 RUN apt-get update && \
-    apt-get upgrade -y && \
+    DEBIAN_FRONTEND=noninteractive apt-get upgrade -y && \
     DEBIAN_FRONTEND=noninteractive apt-get install -y \
-		libssl1.0.0 \
-		ca-certificates \
-		curl && \
+      libssl1.1 \
+      ca-certificates \
+      curl && \
     apt-get autoremove -y && \
-    apt-get clean
-    
-RUN	find /var/lib/apt/lists/ -type f -not -name lock -delete
+    apt-get clean && \
+    find /var/lib/apt/lists/ -type f -not -name lock -delete
 
 COPY ./substrate /usr/local/bin
 
 
-
 RUN useradd -m -u 1000 -U -s /bin/sh -d /substrate substrate
 USER substrate
 
 ENV RUST_BACKTRACE 1
 
+# check if executable works in this container
+RUN /usr/local/bin/substrate --version
+
 EXPOSE 30333 9933 9944
 VOLUME ["/substrate"]
```

Comment:
`VOLUME` actually is not really needed (or I just do not know why is it needed for).
When a container with `VOLUME` defined, but no `docker run ... -v ...` specified in a command; gets launched, it produces an anonymous folder that stores it's data there.
Not many users know about it, so their systems just collect trash over time.

Instead of it, anyone can run `docker run container-name -v "/any/local/dir : /any/container/dir"` and have it locally.

Reply:
:+1: